hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/ExchangeRateManager.java

```diff
@@ -112,25 +126,41 @@ public void update(@NonNull final Bytes bytes) {
         } catch (final IOException e) {
             throw new HandleException(ResponseCodeEnum.INVALID_EXCHANGE_RATE_FILE);
         }
+        System.out.println("proposed rates1 " + proposedRates);
```

Comment:
Let's make sure to remove these printlns.

Reply:
Still there. 🙂

Reply:
I removed it in the other pr test. that still open